frontend/src/utilities/time.ts

```diff
@@ -154,3 +154,27 @@ export const relativeTime = (current: number, previous: number): string => {
 
   return `${date.getDate()} ${monthAsString} ${date.getFullYear()}`;
 };
+
+/** Function to convert time strings like "2Hour" to seconds */
+export const convertPeriodicTimeToSeconds = (timeString: string): number => {
+  const numericValue = parseInt(timeString, 10);
+
+  if (isNaN(numericValue)) {
+    return 0; // return 0 if unit is not recognized
```

Comment:
Descriptive comment for descriptive code... this is unhelpful.

```
// Opens a file
file.open()
```

If your comment is describing static code -- it does not have a lot of value and often times gets ignored during refactors. Recommend avoiding these kind of comments.